src/main/java/org/omnifaces/facesviews/FacesViews.java

```diff
@@ -398,15 +399,26 @@ private static void scanAndStoreWelcomeFiles(ServletContext servletContext) {
 		URL webXml;
 
 		try {
-			webXml = servletContext.getResource("/WEB-INF/web.xml");
+			// default EE location of web.xml
+			webXml = servletContext.getResource(WebXml.WEB_XML);
 		}
 		catch (MalformedURLException e) {
 			throw new IllegalStateException(e);
 		}
 
+		if (webXml == null) {
+			try {
+				// try Quarkus location if not found
+				webXml = servletContext.getResource("/" + WebXml.QUARKUS_WEB_XML);
```

Comment:
This is not correct. The Quarkus one is a classpath resource not a web resource.

Leave this all, I'll fix myself.